### PR TITLE
Small Cleanup

### PR DESCRIPTION
**Pull request by CollinsSpencer**
## Summary

- Updated packages.
- Renamed `/containers` directory to `/pages`.
- Refactored Firebase setup into individual contexts with individual hooks.
- Removed unused Firestore commands.
- Update commands so that the local, emulated Functions are used rather than the production Functions.

## Testing

- Make sure that the app still runs correctly.

### CODE REVIEW

**Review by rrgahan — APPROVED** [review]

**Comment by rrgahan** [comment]
Things looked good, go ahead and merge when able. I'll let you decide how we want to handle synchronizing all the changes that will be coming, if we want to deploy as we go or hold off until everything is built out

**Review by JarodTAerts — APPROVED** [review]
LGTM

**Pull request merged**
